recipes/pynini/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "pynini" %}
+{% set version = "2.0.9.post2" %}
+
+package: 
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: http://www.opengrm.org/twiki/pub/GRM/PyniniDownload/{{ name }}-{{ version }}.tar.gz
+  sha256: 6da376d0d945ec0324b386eb729359c9f0f8e2d8ffa7536c2d23f6d43fb2e3fb
+  
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - openfst ==1.7.5
```

can it be at least 
```suggestion
    - openfst >=1.7.5
```
?

---

Not realistically. Virtually every release of OpenFst involves massive amounts of broken ABI and API compatibility (the versioning is not semantic), and thus each version of Pynini is intended to be used with a single OpenFst release.